testbot.py

```diff
@@ -17,8 +17,10 @@
 from discord.ext import commands
 from os.path import join, getsize
 import re
+
```

Comment:
Amazing addition of a new line here. This improves the program performance by almost 500% it's truly shocking.